app/views/users/new.html.erb

```diff
@@ -1,23 +1,10 @@
-<%= form_for @user do |f| %>
-  <p>
-    <%= f.label :username %>
-    <%= f.text_field :username %>
-  </p>
-
-  <p>
-    <%= f.label :email %>
-    <%= f.text_field :email %>
-  </p>
-
-  <p>
-    <%= f.label :password %>
-    <%= f.password_field :password %>
-  </p>
-  <p>
-    <%= f.label :password_confirmation %>
-    <%= f.password_field :password_confirmation %>
-  </p>
-  <p>
-    <%= f.submit 'Sign up' %>
-  </p>
-<% end %> 
+<% provide(:button_text, 'Sign Up') %>
+<div class="container">
+  <div class="row text-center">
+    <div class="offset-lg-4">
+      <h2>Creating New User</h2>
+      <hr>
+      <%= render 'users/user_form', user: @user %>
```

Comment:
This can just be `render :user_form`.

Since the view you are rendering (new) lives in the users directory, Rails will look in that directory for the partial you are referencing. You only need to define the full path to a partial in 2 circumstances: 

- you are referencing a partial from a partial (nested partials) 
- You are referencing a partial that doesnt live in the same view path as the view you are currently renderign 
  - ie. im rendering /bets/show and I want to render /users/_user_information